ivy/functional/backends/numpy/random.py

```diff
@@ -113,7 +113,7 @@ def randint(
 
 def seed(*, seed_value: int = 0) -> None:
```

Comment:
I think this should be a void function?

Reply:
@KareemMAX, I have made the changes as per the reply from Ved. Please look into it now. Thank you.